### PR TITLE
[WIP] Workload list show workloads from all clusters

### PR DESCRIPTION
**Pull request by hhovsepy**
RFE https://github.com/kiali/kiali/issues/5764

Workloads are listed for namespaces across multiple clusters.
Screenshot from 2023-03-10 18-32-15

### CODE REVIEW

File: business/workloads.go [hunk 1]
```diff
@@ -613,6 +635,8 @@ func fetchWorkloads(ctx context.Context, layer *Layer, namespace string, labelSe
 		return nil, err
 	}
 
+	kialiClient := clientFactory.GetSAClient(cluster)
```
Comment (leandroberetta) [inline business/workloads.go:638]: I think we need to use the user client here?, to just get what the user is allowed to see?.

**Review by leandroberetta — COMMENTED** [review]
First iteration, so far so good, I just have one question below.

File: business/workloads.go (hunk 1 above)
Comment (hhovsepy) [reply business/workloads.go:638]: @leandroberetta thanks, right, will use the user client once https://github.com/kiali/kiali/pull/5851 get merged, for now the main point of this PR is to rework workloads list and details to add "cluster" property.

**Comment by hhovsepy** [comment]
Working on tests.

**Review by josunect — COMMENTED** [review]
I know this is still a draft but I will add some comments (Maybe they were not addressed yet): 

image

- Should we show the cluster column if we are in single cluster? 
- I think we can discuss what name to show for the home cluster (_kiali_home)
- I think we can also add the cluster filter:
image


In the workload details, I have an issue with the graph: 

image

**Comment by hhovsepy** [comment]
Facing some local git issues, closing this PR in a favor to https://github.com/kiali/kiali/pull/5944

**Pull request closed without merging**